app/views/calculators/guaranteed_income/_form.html.erb

```diff
@@ -0,0 +1,38 @@
+<%
+  form = local_assigns[:form]
+  errors = form&.errors
+%>
+
+<h2 id="calculator">Estimate how much your guaranteed income could be</h2>
+
+<%= render 'calculators/error_summary', messages: errors&.messages if form&.invalid? %>
+
+<form action="/guaranteed-income/estimate#form" method="get">
+  <%= render 'calculators/form_input_currency',
+             field: :pot,
+             value: form&.pot,
+             label: 'Amount you\'ll have your pot when you retire',
```

Comment:
"...or shall we assume that the customer is in a position to buy an annuity now in which case we could ask "Amount in your pot" as we do for the other calculators."

This!